### ISSUE TITLE
Changing labels locally

### ISSUE THREAD

**Issue by seifsallam** [fabien-d/alertify.js#205]
I want to have confirm alert when i delete, with label `Delete` as ok. Problem is changing labels alter the global setting, so the next alert will have label `Delete`. Is there anyway I can alter labels without propagating the change globally?

``` coffeescript
alertify.labels.ok = "Delete"

alertify.confirm 'Are you sure you want to delete?', (confirmed) =>
  if confirmed
    # Delete

alertify.labels.ok = "Ok"
```

**Comment by fabien-d** [comment]
Unfortunately not at this time. It may be worth doing in the `0.5.0` [release](https://github.com/fabien-d/alertify.js/pull/173/).

**Comment by seifsallam** [comment]
it would be really awesome. Thanks

**Comment by aneeshbharadwajka** [comment]
Hey was this issue resolved? I am working on the 1.7.1 version and this issue still is present.

Any help would be appreciated.Thanks.

**Comment by reza-aslani** [comment]
It's really necessary.
